datadog_checks_dev/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+***Added***:
+
+* Extend messages from dependency validation. ([#15558](https://github.com/DataDog/integrations-core/pull/15558))
+  1. Say which ddev command to run to fix particular type of error.
+  2. Report success at the end.
```

Comment:
https://github.com/pydantic/pydantic/pull/6408#issuecomment-1620576762

```suggestion
    1. Say which ddev command to run to fix particular type of error.
    2. Report success at the end.
```

Reply:
so the indentation level should be 4 spaces?